### ISSUE TITLE
The floating texts are everywhere on the Server

### ISSUE THREAD

**Issue by Wertzui123** [JackMD/MysteryCrate#94]
I wrote the folder name in the config.yml and restartet my server, but the floating texts are eveywhere on the Server at the Coridinates. In the Console is "Please select the Crateworld" but I did it. Can anyone help me?

**Comment by JackMD** [comment]
Like I mentioned on poggit, it could be that the world might not be loaded. 

The solution for 'world not loaded' is to either use a plugin that loads the worlds on startup or wait for me to update this plugin to load the world mentioned in the config. The latter would take some time since I am busy on far more important stuff than maintaining this plugin. 

As for the floating texts appearing in every world at the coordinates is not new i.e. I know this bug occurs. I used to have checks in this plugin that would remove them from other worlds. A while ago I had to remove it due to a bug in PocketMine which is now addressed/fixed in [cbb9c4f](https://github.com/pmmp/PocketMine-MP/commit/cbb9c4f2989740ca5c1db5445c2f0b7bc0f40410). I will be adding those checks back to stop them from showing on every world. But like I said previously I don't have time for fixing it at the very moment. So it will take some time before its fixed on the plugins side.
